xarray/core/concat.py

```diff
@@ -161,6 +161,45 @@ def concat(
     --------
     merge
     auto_combine
+
+    Examples
+    --------
+    >>> da = xr.DataArray(np.random.randn(2, 3),
+    ...                    [("x", ["a", "b"]),
+    ...                    ("y", [10, 20, 30])])
+    >>> xr.concat([da[:, :1], da[:, 1:]], dim="y")
```

Comment:
Wouldn't `isel` be a better alternative here ?


```suggestion
    >>> xr.concat([da.isel(y=slice(0, 1)), da.isel(y=slice(1, None))], dim="y")
```